lib/SyTest/Identity/Server.pm

```diff
@@ -729,16 +730,16 @@ sub invites_for
    return $self->{invites}{ join "\0", $medium, $address };
 }
 
-=head2
+=head2 get_access_token
 
-   $self->get_access_token();
+   $server->get_access_token();
```

Comment:
I'm not going to make you go through all these yet again, but for future reference: this *is* the example, so you should make it look like what you've written under 'Example' and get rid of the separate example.